### PR TITLE
Update TextLink

### PR DESCRIPTION
**Pull request by DSil**
Fixed colors on active state and also the size of the icons

[ORBIT-2720](https://kiwicom.atlassian.net/browse/ORBIT-2720)

### CODE REVIEW

File: packages/orbit-components/src/TextLink/index.tsx [hunk 1]
```diff
@@ -78,16 +78,23 @@ const getSizeToken = ({ theme, size }: { theme: Theme; size: Common.Size }): str
...
 
+const getIconSize = (theme: Theme, size: Props["size"]) => {
+  if (size === SIZE_OPTIONS.LARGE)
+    return { width: theme.orbit.widthIconLarge, height: theme.orbit.heightIconLarge };
+  if (size === SIZE_OPTIONS.SMALL)
+    return { width: theme.orbit.widthIconSmall, height: theme.orbit.heightIconSmall };
+  return { width: theme.orbit.widthIconMedium, height: theme.orbit.heightIconMedium };
+};
+
 const StyledIconContainer = styled(({ children, className }) => (
```
Comment (mainframev) [inline packages/orbit-components/src/TextLink/index.tsx:89]: I know it was not touched, but I think we can remove here this arrow function with span and have only `styled.span` ;)

File: packages/orbit-components/src/TextLink/index.tsx [hunk 2]
```diff
@@ -78,16 +78,23 @@ const getSizeToken = ({ theme, size }: { theme: Theme; size: Common.Size }): str
   return sizeTokens[size];
 };
 
+const getIconSize = (theme: Theme, size: Props["size"]) => {
```
Comment (mainframev) [inline packages/orbit-components/src/TextLink/index.tsx:81]:
```suggestion
const getIconSize = ({ theme, size }: { theme: Theme, size: Props["size"] }) => {
```

File: packages/orbit-components/src/TextLink/index.tsx [hunk 3]
```diff
@@ -78,16 +78,23 @@ const getSizeToken = ({ theme, size }: { theme: Theme; size: Common.Size }): str
...
 ))`
-  ${({ theme }: { theme: Theme }) => css`
+  ${({ theme, size }: { theme: Theme; size: Props["size"] }) => css`
     display: flex;
     align-items: center;
 
     svg {
-      width: ${theme.orbit.widthIconSmall};
-      height: ${theme.orbit.heightIconSmall};
+      ${getIconSize(theme, size)};
```
Comment (mainframev) [inline packages/orbit-components/src/TextLink/index.tsx:97]:
```suggestion
      ${getIconSize};
```

**Review by mainframev — CHANGES_REQUESTED** [review]
LGTM :) nothing serious, just some small suggestions

File: packages/orbit-components/src/TextLink/index.tsx (hunk 3 above)
Comment (DSil) [reply packages/orbit-components/src/TextLink/index.tsx:97]: I like the previous suggestion but on this one I'd rather do `getIconSize({theme, size})`, like we do on `getSizeToken`

**Review by mainframev — APPROVED** [review]

**Pull request merged**